modules/drivers/velodyne/README_cn.md

```diff
@@ -24,8 +24,8 @@ velodyne驱动是以nodelet的形式实现的，包含了:
 
 ```bash 
 # in dev docker
-cd /apolloe
-bash apollo.sh build_velodyne
+cd /apollo
+bash apollo.sh buildvelodyne
```

Comment:
It is build_velodyne
https://github.com/ApolloAuto/apollo/blob/master/apollo.sh#L650